core/templates/dev/head/components/ck-editor-helpers/ck-editor-4-rte.directive.ts

```diff
@@ -86,8 +89,10 @@ angular.module('oppia').directive('ckEditor4Rte', [
         }).join(',');
         var buttonNames = [];
         names.forEach(function(name) {
-          buttonNames.push('Oppia' + name);
-          buttonNames.push('-');
+          if (!(contextIsExplorationPlayer && name === 'image')) {
```

Comment:
There is still special-casing of 'image' here. I commented on this last time.

Please proofread your PR (go to the "Files Changed" tab) before requesting a review.

Reply:
Sorry, I will take care from the next time